tad/api/routes/tasks.py

```diff
@@ -0,0 +1,48 @@
+from typing import Annotated, Any
+
+from fastapi import APIRouter, Depends, Request, status
+from fastapi.responses import HTMLResponse
+from fastapi.templating import Jinja2Templates
+
+from tad.models.task import MoveTask
+from tad.services.tasks import TasksService
+
+router = APIRouter()
+templates = Jinja2Templates(directory="tad/site/templates")
+
+
+@router.post("/move", response_class=HTMLResponse)
+async def move_task(
```

Comment:
i would make the API name different: see https://cloud.google.com/apis/design/standard_methods and https://cloud.google.com/apis/design/resource_names.

i would make it put or patch /tasks/<id>

Reply:
I like this. I hope it works with different bodies, as this patch on /tasks will update the position using a MovedTask body. We probably will get other bodies in the future. We'll see.